### PR TITLE
fix: Unable to access service through ingress

### PR DESCRIPTION
**Pull request by CloudsLetter**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: helm/charts/kratos/templates/ingress-admin.yaml [hunk 1]
```diff
@@ -42,7 +42,7 @@ spec:
               service:
                 name: {{ $fullName }}-admin
                 port:
-                  name: http
+                  name: http-admin
               {{- else }}
               serviceName: {{ $fullName }}-admin
               servicePort: http
```
Comment (Demonsthere) [inline helm/charts/kratos/templates/ingress-admin.yaml:48]:
```suggestion
              servicePort: http-admin
```

File: helm/charts/kratos/templates/ingress-public.yaml [hunk 2]
```diff
@@ -42,7 +42,7 @@ spec:
               service:
                 name: {{ $fullName }}-public
                 port:
-                  name: http
+                  name: http-public
               {{- else }}
               serviceName: {{ $fullName }}-public
               servicePort: http
```
Comment (Demonsthere) [inline helm/charts/kratos/templates/ingress-public.yaml:48]:
```suggestion
              servicePort: http-public
```

**Review by Demonsthere — COMMENTED** [review]
Hi there,
Good catch! Must have missed that when refactoring services 😓

**Review by Demonsthere — APPROVED** [review]

**Comment by CloudsLetter** [comment]
> Hi there, Good catch! Must have missed that when refactoring services 😓
Yes, I forgot to modify it :)

**Pull request merged**